kairon/chat/handlers/channels/business_messages/business_messages.py

```diff
@@ -0,0 +1,187 @@
+import hmac
+import uuid
+import base64
+import hashlib
+
+from typing import Text, Dict, Any, Optional
+from starlette.requests import Request
+from loguru import logger
+from rasa.core.channels.channel import UserMessage, InputChannel
+
+from businessmessages import businessmessages_v1_client as bm_client
+from businessmessages.businessmessages_v1_messages import (
+    BusinessmessagesConversationsMessagesCreateRequest, BusinessmessagesConversationsEventsCreateRequest,
+    BusinessMessagesEvent
+)
+from businessmessages.businessmessages_v1_messages import BusinessMessagesMessage
+from businessmessages.businessmessages_v1_messages import BusinessMessagesRepresentative
+from oauth2client.service_account import ServiceAccountCredentials
+
+from kairon.shared.models import User
+from kairon.shared.chat.processor import ChatDataProcessor
+from kairon.chat.agent_processor import AgentProcessor
+
+
+path_to_service_account_key = 'kairon/chat/handlers/channels/business_messages/service_account_key.json'
```

Comment:
remove the part where we are reading credentials twice from file on disk and directly utilise the dict to create credentials